client/state/login/actions.js

```diff
@@ -52,6 +52,21 @@ import wpcom from 'lib/wp';
 import { addLocaleToWpcomUrl, getLocaleSlug } from 'lib/i18n-utils';
 import { recordTracksEventWithClientId as recordTracksEvent } from 'state/analytics/actions';
 
+/**
+ * Fetch all remote login urls
+ *
+ * @param  {Array}   loginLinks     Array of urls
+ * @return {Promise}                A promise that always resolve
+ */
+export const remoteLoginUser = loginLinks => {
+	return Promise.all(
+		loginLinks
+			.map( loginLink => fetch( loginLink ) )
```

Comment:
> By default, fetch won't send or receive any cookies from the server, resulting in unauthenticated requests if the site relies on maintaining a user session (to send cookies, the credentials init option must be set).
Since Aug 25, 2017. The spec changed the default credentials policy to same-origin. Firefox changed since 61.0b13.

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

So I think we should pass `credentials: include` here